css/site.styl

```diff
@@ -345,6 +346,24 @@ img
 
 // HEADER
 
+#header 
+  text-align: center
+
+.menu-item
+  margin: 0 $margin-tiny
+  letter-spacing: .1em
+  
+.editorial-title
```

Comment:
there is a utility class for extra letter spacing that possibly could be used here